src/v2/Assets/Colors.tsx

```diff
@@ -18,4 +18,5 @@ export default {
   yellowBold: "#f1af1b",
   greenRegular: "#0eda83",
   greenBold: "#0aa462",
+  greenToast: "#127438",
```

Comment:
This file has been deprecated in favor of Palette, and we can pull values out via our `useTheme` hook:
```tsx
import { useTheme } from '@artsy/palette'

... 

const { theme } = useTheme()
console.log(theme.colors)
```

All available design-system colors can be seen [here](https://palette.artsy.net/tokens/color)